x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.test.ts

```diff
@@ -59,61 +59,5 @@ it('getWaterfallRoot', () => {
     }
   ];
 
-  expect(getWaterfallRoot(items, items[4])).toEqual({
-    duration: 9480,
-    docType: 'transaction',
-    id: 'a',
-    name: 'APIRestController#products',
-    offset: 0,
-    serviceName: 'opbeans-java',
-    timestamp: 1536763736366000,
-    children: [
-      {
-        duration: 4694,
-        docType: 'span',
-        id: 'b2',
-        name: 'GET [0:0:0:0:0:0:0:1]',
-        offset: 1000,
-        parentId: 'a',
-        serviceName: 'opbeans-java',
-        timestamp: 1536763736367000,
-        children: []
-      },
-      {
-        duration: 4694,
-        docType: 'span',
-        id: 'b',
-        name: 'GET [0:0:0:0:0:0:0:1]',
-        offset: 2000,
-        parentId: 'a',
-        serviceName: 'opbeans-java',
-        timestamp: 1536763736368000,
-        children: [
-          {
-            duration: 3581,
-            docType: 'transaction',
-            id: 'c',
-            name: 'APIRestController#productsRemote',
-            offset: 3000,
-            parentId: 'b',
-            serviceName: 'opbeans-java',
-            timestamp: 1536763736369000,
-            children: [
-              {
-                duration: 210,
-                docType: 'span',
-                id: 'd',
-                name: 'SELECT',
-                offset: 5000,
-                parentId: 'c',
-                serviceName: 'opbeans-java',
-                timestamp: 1536763736371000,
-                children: []
-              }
-            ]
-          }
-        ]
-      }
-    ]
-  });
+  expect(getWaterfallRoot(items, items[4])).toMatchSnapshot();
```

Comment:
This felt like a really natural place to use a snapshot. :)

Reply:
Hmm.. I actually find that snapshots make it harder to read tests. I mostly use them for component.

Reply:
Cool, I think once it becomes a giant object that is too hard to visually grok, snapshots become super useful because of the diff output.